docs/_data/portland-2022-config.yaml

```diff
@@ -69,11 +69,11 @@ buttons:
 
 tickets:
   corporate:
-    price: $500
+    price: $200
```

Comment:
Just double-checking, these are the same prices as last year right? TBH I don't see a need to raise prices so if they're the same then it's nice and friendly :)

Reply:
Yea, we can even mention "not impacted by inflation" :)